plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -104,6 +104,8 @@ class read_only {
       //string                  recent_slots;
       //double                  participation_rate = 0;
       optional<string>        server_version_string;
+      uint32_t                fork_db_head_block_num = 0;
+      chain::block_id_type    fork_db_head_block_id;
    };
```

Comment:
Need to be optional to allow new cleos to work with old nodeos as cleos main.cpp uses `call(url, get_info_func).as<eosio::chain_apis::read_only::get_info_results>()`